layouts/shortcodes/board-carousel.html

```diff
@@ -79,6 +79,12 @@
         <p>BeagleBone AI-64</p>
       </a>
     </li>
+    <li id="c1_slide14">
```

Comment:
You need to adjust the the following numbers so that they're in order. And then please also add it to the list in lines 100 - 117.